mkdocs.yml

```diff
@@ -0,0 +1,31 @@
+# Project information
+site_name: "Uganda EMR Technical Guide"
+site_description: "Technical documentation for Uganda EMR"
+site_author: "METS"
+site_url: "https://github.com/intrahealth/client-registry"
+
+# Repository
+repo_name: "ugandaemr-technicalguide"
+repo_url: "https://austinbwanika.github.io/ugandaemr-technicalguide/"
```

Comment:
@AustinBwanika The repo URL should be https://github.com/METS-Programme/ugandaemr-technicalguide